addons/purchase/models/purchase.py

```diff
@@ -612,33 +421,6 @@ def _compute_qty_invoiced(self):
                         qty -= inv_line.uom_id._compute_quantity(inv_line.quantity, line.product_uom)
             line.qty_invoiced = qty
 
-    @api.model
-    def create(self, values):
-        line = super(PurchaseOrderLine, self).create(values)
-        if line.order_id.state == 'purchase':
```

Comment:
@jem-odoo Maybe, this information could be also useful for service. You could maybe keep it. As you prefer for me.

Reply:
Good idea, for sale order, the message_post is done in sale module. SO I'll move this to purchase.